device_doctor/lib/src/ios_debug_symbol_doctor.dart

```diff
@@ -50,8 +50,25 @@ class DiagnoseCommand extends Command<bool> {
       return false;
     }
 
+    await checkDevToolsSecurity();
+
     return true;
   }
+
+  /// Log the status of DevToolsSecurity.
+  Future<void> checkDevToolsSecurity() async {
+    final List<String> command = <String>['xcrun', 'DevToolsSecurity', '--status'];
+    final io.ProcessResult result = await processManager.run(
+      command,
+    );
+    if (result.exitCode != 0) {
+      logger.severe(
```

Comment:
is your intention for this just to be diagnostic, or should this block the build?

Reply:
Diagnostic, does `severe` block the build?

Reply:
nope it doesn't. if this is diagnostic only then this approach looks fine